plugins/labhub.py

```diff
@@ -239,7 +247,13 @@ def newcomer_difficulty_level(user, iss):
             2. A newcomer asks for assignment to an issue with difficulty higher
                than low.
             """
-            if self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user):
+            if (self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user) and
```

Comment:
I guess, since it checks for all statures, we rename it to just `difficulty_level`